data/repolist.txt

```diff
@@ -165,6 +166,7 @@ dotnet/sdk branch=release/2.1.1xx server=dotnet-ci
 dotnet/sdk branch=release/2.1.2xx server=dotnet-ci
 dotnet/sdk branch=release/2.1.3xx server=dotnet-ci
 dotnet/sdk branch=release/2.1.4xx server=dotnet-ci
+dotnet/sdk branch=release/2.2.1xx server=dotnet-ci
```

Comment:
not just 2.2 or `2.2.0xx`?

Reply:
No. We explicitly want it to be 1xx, to differentiate from the runtime. We want to prevent people thinking the numbers will stay aligned.